python_modules/dagster/dagster/_core/execution/plan/compute.py

```diff
@@ -142,7 +138,7 @@ def _yield_compute_results(
     step_context: StepExecutionContext,
     inputs: Mapping[str, Any],
     compute_fn: OpComputeFunction,
-    compute_context: Union[OpExecutionContext, AssetExecutionContext, AssetCheckExecutionContext],
+    compute_context: ExecutionContextTypes,
```

Comment:
`ExecutionContextTypes` is `Union[OpExecutionContext, AssetExecutionContext, AssetCheckExecutionContext]` just changing it here to make the type signature more concise